### PR TITLE
Placements

### PR DESCRIPTION
**Pull request by wasade**
Add support for handling jplace fragments

@kwcantrell, any concerns?

@antgonza, in our test data which used a 10k backbone with 1M placements, this took about 100 seconds total. Any chance you could pull down and see how this behaves with placement data from Qiita?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,29 @@ problem of requiring numpy and cython for setup.py to execute. The package is
...
+```
+$ bp placement --help
+Usage: bp placement [OPTIONS]
+
+Options:
+  --placements PATH               jplace formatted data  [required]
+  --output PATH                   Where to write the resulting newick
+                                  [required]
+
+  --method [fully-resolved|multifurcating]
```
Comment (antgonza) [inline README.md:35]: I'm wondering if it somewhere in the documentation the difference between the options in this parameter should be explained. To be honest, I don't know the difference.

**Review by antgonza — COMMENTED** [review]
Thank you for working on this; mainly comments about the install instructions and some documentation.

File: README.md [hunk 2]
```diff
@@ -12,6 +12,29 @@ problem of requiring numpy and cython for setup.py to execute. The package is
 named iow in pypi as "bp" was taken at time of registration.
 
 ```
-conda install numpy cython
-pip install iow
+$ conda install numpy cython
```
[the remark this answers is not in the document]
Comment (wasade) [reply README.md:15]: Thanks! Normally this is just installed with "pip install iow" but i'll add a section for development use

File: README.md (hunk 1 above)
Comment (wasade) [reply README.md:35]: kk

File: README.md [hunk 3]
```diff
@@ -12,6 +12,29 @@ problem of requiring numpy and cython for setup.py to execute. The package is
...
+Options:
+  --placements PATH               jplace formatted data  [required]
+  --output PATH                   Where to write the resulting newick
+                                  [required]
+
+  --method [fully-resolved|multifurcating]
+                                  Whether to fully resolve or multifurcate
+                                  [required]
+
+  --help                          Show this message and exit.
```
[the remark this answers is not in the document]
Comment (wasade) [reply README.md:39]: kk

File: README.md (hunk 1 above)
Comment (wasade) [reply README.md:35]: actually, there is a description on line 22, is that good enough?

**Comment by antgonza** [comment]
Thank you for adding the new documentation and notes; I think is ready to be merge - however, I don't have merge permissions in this repo ...

**Comment by wasade** [comment]
Thanks!!

**Comment by wasade** [comment]
Before release, I need to add in the requested changes on fragments with multiple placements -- more later today.

**Pull request merged**